doc/how-to-analyze-haskell-code-performance.rst

```diff
@@ -0,0 +1,126 @@
+How to analyze Haskell performance
+==================================
+
+When a Haskell application is slow or uses too much memory,
+Cabal and `GHC <https://downloads.haskell.org/ghc/latest/docs/users_guide/>`__
```

Comment:
Should this point to the [profiling section](https://downloads.haskell.org/ghc/latest/docs/users_guide/profiling.html) of the GHC manual?

Reply:
done